<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelectWidget</class>
 <widget class="QWidget" name="ImageSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QListView" name="fileList"/>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="fromFileButton">
     <property name="text">
      <string>From File:</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="filenameField"/>
   </item>
   <item row="2" column="0" alignment="Qt::AlignTop">
    <widget class="QRadioButton" name="fromMemoryButton">
     <property name="text">
      <string>From Memory:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QRadioButton" name="noneButton">
     <property name="text">
      <string>None</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
